src/app/app.routes.ts

```diff
@@ -16,6 +17,7 @@ export const routes: Routes = [
   { path: 'companyHome', component: CompanyHomeComponent },
   { path: 'customerHome', component: CustomerHomeComponent },
   { path: 'addCustomerForm', component: AddCustomerFormComponent },
+  { path: 'updateCustomerForm', component: UpdateCustomerFormComponent},
```

Comment:
change to 'updateCustomerForm/id'
because you want to transfer the customer's id who need to update